ui/dashboards/src/components/AddPanel/AddPanel.tsx

```diff
@@ -41,26 +44,42 @@ const AddPanel = ({ isOpen, onClose }: AddPanelProps) => {
   const [group, setGroup] = useState(0);
   const [panelName, setPanelName] = useState('');
   const [panelDescription, setPanelDescription] = useState('');
+  const [kind, setKind] = useState('');
+  const [options, setOptions] = useState<JsonObject>({});
 
-  const onSelectGroupChange = (e: SelectChangeEvent<number>) => {
-    setGroup(e.target.value as number);
+  const handleGroupChange: SelectProps<number>['onChange'] = (e) => {
```

Comment:
Should we type the `e` argument like the other handlers below?

Reply:
Good question. If you type the handler like I did here (e.g. `const handleGroupChange: SelectProps<number>['onChange']`) Typescript takes care of inferring the parameter(s). I tend to like doing handlers this way so that the definition for the props is "controlling" the type of the parameters and if they ever change the handler definition, you don't have to back and redefine a bunch of argument types.

Reply:
Ah yes, that sounds nicer